tools/android_sdk_packages.txt

```diff
@@ -0,0 +1,5 @@
+platforms;android-31:platforms
```

Comment:
nice. minor nit: what about moving to a folder. Maybe `/tools/android_sdk` then this file becomes `packages.txt`?